File: Mad_libs_game.c
/*
|Author : Mazen Sakr
|File  Name : Mad libs game
|Description :-
*/

//  Includes section
#include<stdio.h>
#include<stdlib.h>
/*********************/

void main(void) {
    system("color 30");
    char adjective1[20];
    char noun1[20];
    char verb1[20];
    char adverb[20];
    char adjective2[20];
    char noun2[20];
    char noun3[20];
    printf("This is a mad lips game \nfill in the blanks in the following text in order\nToday I went to the zoo.I saw a(n)___________(adjective)\n_____________(Noun) jumping up and down in its tree.\nHe _____________(verb, past tense) __________(adverb)\nthrough the large tunnel that led to its _______(adjective)\n__________(noun).I got some peanuts and passed\nthem through the cage to a gigantic gray _______(noun)\ntowering above my head.");
    printf("\nAdjective 1 is :");
    scanf("%s", &adjective1);
    printf("\nNoun 1 is :");
    scanf("%s", &noun1);
    printf("\nVerb 1 is :");
    scanf("%s", &verb1);
    printf("\nAdverb is :");
    scanf("%s", &adverb);
    printf("\nAdjective 2 is :");
    scanf("%s", &adjective2);
    printf("\nNoun 2 is :");
    scanf("%s", &noun2);
    printf("\nNoun 3 is :");
    scanf("%s", &noun3);
    printf("\nThe text is :\nToday I went to the zoo.I saw a(n) %s\n%s jumping up and down in its tree.\nHe %s %s\nthrough the large tunnel that led to its %s\n%s.I got some peanuts and passed\nthem through the cage to a gigantic gray %s\ntowering above my head.",adjective1,noun1,verb1,adverb,adjective2,noun2,noun3);
}

File: MadLibs_Game.py
#---------------------------------------------------------------------------
# file : Madlibs
# Author : mazen sakr
#-------------------------------------------------------------

#main code :
print("This is a madlibs game, Please input words into the empty spaces\n")
print("""Today I went to the zoo. I saw a(n)
___________(adjective)\n
_____________(Noun) jumping up and down in its tree.\n
He _____________(verb, past tense) __________(adverb)\n
through the large tunnel that led to its _______(adjective)\n
__________(noun). I got some peanuts and passed\n
them through the cage to a gigantic gray _______(noun)\n
towering above my head.\n""")
Adjective1 = input ("Please input the first adjective \n")
Noun1 = input ("Please input the first noun \n")
Verb1 = input ("Please input the first verb \n")
Adverb = input ("Please input the adverb \n")
Adjective2 = input ("Please input the second adjective \n")
Noun2 = input ("Please input the secod noun \n")
Noun3 = input ("Please input the third noun \n")
print("Today I went to the zoo. I saw a(n)"+Adjective1+"\n"+Noun1+"jumping up and down in its tree.\nHe"+Verb1+Adverb+"""\n
through the large tunnel that led to its"""+Adjective2+"\n"+Noun2+""". I got some peanuts and passed\n
them through the cage to a gigantic gray"""+Noun3+"\ntowering above my head.\n")